content/800-guides/330-management-api-basic.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: 'Get started with the Prisma Management API'
+metaTitle: "Get started with the Prisma Management API"
+description: "Learn how to get started with the Prisma Management API"
+sidebar_label: 'Management API starter'
+image: '/img/guides/neon-connection-pooling.png'
+completion_time: '15 min'
+community_section: true
+---
+
+## Overview
+
+This guide walks you through setting up a basic TypeScript project that uses the [Prisma Management API](/postgres/database/api-reference/management-api) to create a new project and database, and print out all connection details. You'll authenticate via a [service token](/postgres/database/api-reference/management-api#bearer-tokens), set up your environment, and run a script to interact with the API.
```

Comment:
```suggestion
This guide walks you through setting up a basic TypeScript script that uses the [Prisma Management API](/postgres/database/api-reference/management-api) to create a new project and database, and print out all connection details. You'll authenticate via a [service token](/postgres/database/api-reference/management-api#bearer-tokens), set up your environment, and run the script to interact with the API.
```

I assumed "project" was an app somehow, and I was a bit confused until reading to the end

Reply:
Ah good feedback, I'll clarify this!